hack/trivy_scan.py

```diff
@@ -148,19 +151,19 @@ def extract_images(version):
                 scan_data = json.load(json_file)
 
             if not scan_data.get('Results'):
-                print(f"No vulnerabilities found in {image_name}:{image_tag}", flush=True)
+                print(f"No vulnerabilitiy found in {image_name}:{image_tag}", flush=True)
             else:
-                vulnerabilities = [
+                vulnerabilitiy = [
                     result['Vulnerabilities']
                     for result in scan_data['Results']
                     if 'Vulnerabilities' in result and result['Vulnerabilities']
                 ]
 
-                if not vulnerabilities:
+                if not vulnerabilitiy:
                     print(f"The vulnerability detection may be insufficient because security updates are not provided for {image_name}:{image_tag}", flush=True)
                 else:
                     severity_counts = {"LOW": 0, "MEDIUM": 0, "HIGH": 0, "CRITICAL": 0}
-                    for vulns in vulnerabilities:
+                    for vulns in vulnerabilitiy:
```

Comment:
"for vulnerability in vulnerabilities:" could make more sense. Or is the highest level a vulnerabilities_map ?